CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Changed
```

Comment:
Do you think this should be noted as "breaking" given the potential impact on callers?

Reply:
IMO, it might be better to describe how an app might be broken by this change and why we did it.

Might not be the best habit to get into to start including a regular "Breaking" section in the semver doc heh.